packages/playground/src/utils/validators.ts

```diff
@@ -733,16 +733,6 @@ export function pattern(msg: string, config: RegexPattern) {
   };
 }
 
-/**
- * Checks if a given string is a valid SSH key.
- * @param key - The SSH key string to be validated.
- * @returns True if the key is a valid SSH key, false otherwise.
- */
-export function isValidSSHKey(key: string): boolean {
-  const sshKeyRegex = /^(ssh-rsa|ssh-dss|ecdsa-[a-zA-Z0-9-]+|ssh-ed25519)\s+(\S+)\s+(\S+)/;
-  return sshKeyRegex.test(key);
-}
-
```

Comment:
why did we move the validation, Shouldn't all validators be here ?

Reply:
It moved to be inside the class management, I want to group all ssh-related things in one sope